packages/fx-core/src/component/driver/util/utils.ts

```diff
@@ -26,3 +26,7 @@ export function mapStateToEnv(
   }
   return result;
 }
+
+export function isNullOrUndefined(value: any): boolean {
```

Comment:
it seems that `null == undefined` is true
https://stackoverflow.com/questions/2647867/how-can-i-determine-if-a-variable-is-undefined-or-null

Reply:
Thanks. The logic comes from https://nodejs.org/api/util.html#utilisnullorundefinedobject. I will keep current implementation, so people do not need to have strict understand of `==` operator in js.

Reply:
I used another method to fix the bug and the code is no longer needed